packages/legend-studio/src/stores/LegendStudioPlugin.ts

```diff
@@ -68,10 +69,20 @@ export type ModelLoaderExtensionConfiguration = {
   renderer: (editorStore: EditorStore) => React.ReactNode | undefined;
 };
 
-export type TestableMetadataGetter = (
+export type TestableElementEditorOpener = (
   testable: Testable,
   editorStore: EditorStore,
-) => TestableMetadata | undefined;
+) => TestableElementEditorState | undefined;
+
+export type TestableExtension = {
+  metadata: TestableMetadata;
+  testableEditorOpener?: TestableElementEditorOpener;
+};
+
+export type TestableExtensionGetter = (
```

Comment:
--> `TestableSupportGetter`, sorry but I think the word extension is a bit broad for just features scoped by `Studio`